Dockerfile

```diff
@@ -1,25 +1,71 @@
-FROM tomcat:jdk11-adoptopenjdk-openj9
+FROM ibm-semeru-runtimes:open-17.0.6_10-jre
 LABEL maintainer="petr@wultra.com"
 
 # Prepare environment variables
-ENV JAVA_HOME /opt/java/openjdk
-ENV TOMCAT_HOME /usr/local/tomcat
-ENV WAR_VERSION 1.4.0
+ENV JAVA_HOME=/opt/java/openjdk \
+    LB_HOME=/usr/local/liquibase \
+    LB_VERSION=4.21.1 \
+    PKG_RELEASE=1~jammy \
+    TOMCAT_HOME=/usr/local/tomcat \
+    TOMCAT_MAJOR=10 \
+    TOMCAT_VERSION=10.1.7 \
+    TZ=UTC
+
+ENV PATH=$PATH:$LB_HOME:$TOMCAT_HOME/bin
+
+# Init
+RUN apt-get -y update  \
+    && apt-get -y upgrade \
+    && apt-get -y install bash curl wget
+
+# Install tomcat
+RUN curl -jkSL -o /tmp/apache-tomcat.tar.gz http://archive.apache.org/dist/tomcat/tomcat-${TOMCAT_MAJOR}/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz \
+    && [ "41997f90baf86af6dc0396cbf66941f801e0ae9ad1b57475661d10e648c55559491e85468af2df3ee457be3fcae12b74537ce19a9a28e34030b98e8bb38dbd35  /tmp/apache-tomcat.tar.gz" = "$(sha512sum /tmp/apache-tomcat.tar.gz)" ] \
+    && gunzip /tmp/apache-tomcat.tar.gz \
+    && tar -C /opt -xf /tmp/apache-tomcat.tar \
+    && ln -s /opt/apache-tomcat-$TOMCAT_VERSION $TOMCAT_HOME
 
 # Clear root context
 RUN rm -rf $TOMCAT_HOME/webapps/*
 
+# This setup was inspired by https://github.com/mobtitude/liquibase/blob/master/Dockerfile
+RUN set -x \
+    && wget -q -O /tmp/liquibase.tar.gz "https://github.com/liquibase/liquibase/releases/download/v$LB_VERSION/liquibase-$LB_VERSION.tar.gz" \
+    && [ "c04542865e5ece8b7b1ee9bd6beaefc5315e350620288d6ac1a2d32c3b1f7d8b  /tmp/liquibase.tar.gz" = "$(sha256sum /tmp/liquibase.tar.gz)" ] \
+    && mkdir -p "$LB_HOME" \
+    && tar -xzf /tmp/liquibase.tar.gz -C "$LB_HOME" \
+    && rm -rf "$LB_HOME/sdk" \
+# Uninstall packages which are no longer needed and clean apt caches
+    && apt-get -y remove wget curl gettext-base \
+    && apt-get -y purge --auto-remove \
+    && rm -rf /tmp/* /var/cache/apt/*
+
+# Liquibase - changesets
+RUN rm -rf $LB_HOME/data
+COPY docs/db/changelog $LB_HOME/db/changelog
+
 # Add valve for proxy with SSL termination
 RUN sed -i 's/<\/Host>/<Valve className="org.apache.catalina.valves.RemoteIpValve" remoteIpHeader="X-Forwarded-For" protocolHeader="X-Forwarded-Proto"\/><\/Host>/' $TOMCAT_HOME/conf/server.xml
 
+# Copy libraries
+COPY enrollment-server/target/dependency/postgresql*.jar $TOMCAT_HOME/lib/
```

Comment:
Described at https://github.com/wultra/enrollment-server/pull/727/files#diff-ccaf5112b753fd0510cb1808a6e09d84aa1be36a93614d2e77a72afead8bd36fR37 But as we have discussed, it will change soon, when we include that dependency.

```shell
mvn dependency:copy-dependencies -Pstandalone
```